lib/kube/kubeconfig/kubeconfig.go

```diff
@@ -359,9 +367,46 @@ func SelectContext(teleportCluster, kubeCluster string) error {
 	if _, ok := kc.Contexts[kubeContext]; !ok {
 		return trace.NotFound("kubeconfig context %q not found", kubeContext)
 	}
+	setSelectedExtension(kc.Contexts, kc.CurrentContext, teleportCluster)
 	kc.CurrentContext = kubeContext
 	if err := Save("", *kc); err != nil {
 		return trace.Wrap(err)
 	}
 	return nil
 }
+
+const selectedExtension = "selected_context"
```

Comment:
Is the name standard? Should we prefix it with `teleport` or something to ensure no other app will rely on the same extension?

Reply:
done